tests/test_accuracy-curve.py

```diff
@@ -45,13 +102,17 @@ def test_correct_num_input():
       
 # Test for correct error handing when arrays are
 # of unequal length
-def test_equal_input_len():
+def test_unequal_input_len():
     with pytest.raises(ValueError):
-        accuracy_curve(train_scores_4, cv_scores_3, param_grid_5)
+        accuracy_curve(train_scores, cv_scores_4, param_grid_4)
 
-# Test for correct error handing when arrays are
-# empty
+# Test for correct error handing when length of arrays is
+# NOT greater than 1
 def test_empty_arrays():
     with pytest.raises(ValueError):
         accuracy_curve(train_scores_empty, cv_scores, param_grid)
         accuracy_curve(train_scores, cv_scores, param_grid_empty)
+        accuracy_curve(train_scores_empty, cv_scores_empty, param_grid_empty)
+        accuracy_curve(train_scores_1, cv_scores_1, param_grid_1)
```

Comment:
If both of these should raise value errors, I think you need to include this in another condition (Otherwise the test will pass even if the second line doesn't raise a value error)

Reply:
ohhhh very good to know! I can change that!

Reply:
I separated the following tests:
```
def test_all_empty_arrays():
    with pytest.raises(ValueError):
        accuracy_curve(train_scores_empty, cv_scores_empty, param_grid_empty)

def test_all_arrays_len1():
    with pytest.raises(ValueError):
        accuracy_curve(train_scores_1, cv_scores_1, param_grid_1)
```

and added this to the accuracy function:
```
if len(train_scores) <= 1 or len(cv_scores) <= 1 or len(param_grid) <=1:
        raise ValueError("Input arrays must have more than one element.")
```
tests run successfully:
```
$ pytest test_accuracy-curve.py 
================================================================== test session starts ===================================================================
platform darwin -- Python 3.11.6, pytest-7.4.3, pluggy-1.3.0
rootdir: /Users/hinabandukwala/Documents/MDS/block_3/522/wine-origin-prediction/tests
collected 11 items                                                                                                                                       

test_accuracy-curve.py ...........                                                                                                                 [100%]

=================================================================== 11 passed in 1.15s ===================================================================
```